CONTRIBUTING.md

```diff
@@ -1,6 +1,6 @@
 # Contributing
 
-- Your favourite command isn't covered?
+- Your favorite command isn't covered?
```

Comment:
Don't worry about it. I don't think I have noticed other words with the US spelling, but not worth having a debate about. Happy to leave `favorite` as you have in the PR.